stable/sonatype-nexus/values.yaml

```diff
@@ -130,3 +130,19 @@ ingress:
     secretName: nexus-tls
 
 tolerations: []
+
+deployment:
```

Comment:
Same comment here. An if block for when config.enabled=true seems like it would be a better end-user experience

Reply:
You are right. I've updated the code accordingly.